doc/source/example_notebooks/IS2_data_read-in.ipynb

```diff
@@ -183,7 +233,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "path_root = '/full/path/to/your/data/'"
+    "path_root = '/Users/jessica/computing/icepyx/github_repo/icepyx/doc/source/example_notebooks/atl11/'"
```

Comment:
Is this hard-coded path intended or just an artifact of running it locally?

Reply:
Great catch - I will fix it!